library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -367,17 +367,17 @@ private boolean updateLocked() {
 
             Intent intent = new Intent(Constants.ACTION_RETRY);
             intent.setClass(this, DownloadReceiver.class);
-            mAlarmManager.set(AlarmManager.RTC_WAKEUP, now + nextRetryTimeMillis, PendingIntent.getBroadcast(this, 0, intent, PendingIntent.FLAG_ONE_SHOT));
+            alarmManager.set(AlarmManager.RTC_WAKEUP, now + nextRetryTimeMillis, PendingIntent.getBroadcast(this, 0, intent, PendingIntent.FLAG_ONE_SHOT));
         }
 
         return isActive;
     }
 
     private void updateTotalBytesFor(DownloadInfo info) {
-        if (info.mTotalBytes == -1) {
+        if (info.getTotalBytes() == -1) {
```

Comment:
Maybe using `!info.hasTotalBytes()` or create the opposite method and use that , although I don't really know how to name it (`hasNotTotalBytes` ?).